code/modules/emotes/definitions/_species.dm

```diff
@@ -1,5 +1,15 @@
 /datum/species
 	var/list/default_emotes = list()
+	// Pulse modifiers
```

Comment:
this is the place to define emotes, you would want to put these vars in
![image](https://user-images.githubusercontent.com/22774890/117834199-40073a80-b277-11eb-979d-08970ca891fa.png)